.github/workflows/deployment.yml

```diff
@@ -28,6 +28,31 @@ jobs:
           config-file: release-please-config.json
           manifest-file: .release-please-manifest.json
 
+  build-deploy-library-package:
+    runs-on: ubuntu-latest
+    needs: release-please
+    if: ${{ github.event_name == 'push' && github.repository == 'DIRACGrid/diracx-web' && needs.release-please.outputs.release_created == 'true' }}
+    permissions:
+      contents: read
+      packages: write
+    steps:
+    - uses: actions/checkout@v4
+
+    - uses: actions/setup-node@v4
+      with:
+        node-version: 'latest'
+        registry-url: 'https://registry.npmjs.org'
```

Comment:
setup-node doesn't seem to login if the registry-url isn't set.
Workflow failed when I tested without it (see [the run](https://github.com/Loxeris/diracx-web/actions/runs/9953633032/job/27497517899)).

Reply:
Cache is handled a bit differently in case we use a monorepo: see https://github.com/actions/setup-node?tab=readme-ov-file#caching-global-packages-data

Could you adapt the github actions workflow accordingly please?

Reply:
Apparently, this isn't needed in our case. The cache leverages the `package-lock.json` file, which is centralized in the root directory for npm workspaces.
It would however be necessary if we used a package not managed by npm workspaces in the github action.

Reply:
Good point!
By the way, at some point, it would be great to run the tests of the extension in the CI too, to make sure it remains up to date with the library.
I guess it's better if the extension is managed separately from the rest of the code. Any opinion?

Reply:
Yeah I think it should be managed separately for now.
It shouldn't be too hard if we want to change that later.